integrations/to.etc.domui.selenium/src/main/java/to/etc/domui/webdriver/core/base/BasePO.java

```diff
@@ -0,0 +1,21 @@
+package to.etc.domui.webdriver.core.base;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import to.etc.domui.webdriver.core.WebDriverConnector;
+
+@NonNullByDefault
+public abstract class BasePO {
```

Comment:
Test proxies for IControl<T> components should have a common interface, with common methods that are possible on them. like isDisabled() and perhaps even get/setValue.